Breakout functions (#47)
datacarpentry/astronomy-python: _episodes/06-photo.md

> * Extinction av = 0
{: .callout}

## Making a polygon
The MIST isochrone files available on the website above can't be directly plotted over our data. 

> We've selected the relevant part of the isochrone, the filter's we're interested, and scaled the photometry to the distance of GD-1 

Reviewer: ```suggestion
We have selected the relevant part of the isochrone, the filters we are interested in, and scaled the photometry to the distance of GD-1 
```

![Right ascension and declination of selected stars in GD-1 frame after selecting for both proper motion and photometry.](../fig/06-photo_files/06-photo_93_0.png)

> This example includes the new Matplotlib command `figure` which creates the larger canvas the the subplots are placed on.  In previous examples, we didn't have

Reviewer: ```suggestion
This example includes the new Matplotlib command `figure`, which creates the larger canvas that the subplots are placed on.  In previous examples, we didn't have
```
